src/deploy/osp_deployer/director.py

```diff
@@ -1656,28 +1677,28 @@ def configure_tempest(self):
         setts = self.settings
         external_sub_guid = self.get_sanity_subnet()
         if not external_sub_guid:
-            err = ("Could not find public network, please run the "
-                   + "sanity test to create the appropriate networks "
-                   + "and re-run this script with the "
-                   + "--tempest_config_only flag.")
+            err = ("Could not find public network, please run the " +
+                   "sanity test to create the appropriate networks " +
+                   "and re-run this script with the " +
+                   "--tempest_config_only flag.")
             raise AssertionError(err)
 
         self._backup_tempest_conf()
 
-        cmd_route = ("source ~/" + setts.overcloud_name + "rc;"
-                     "sudo ip route add " + setts.floating_ip_network
-                     + " dev eth0")
-        cmd_config = ("source ~/" + setts.overcloud_name + "rc;"
-                      "if [ ! -d " + self.tempest_directory
-                      + " -o -z '$(ls -A " + self.tempest_directory
-                      + " 2>/dev/null)' ]; then tempest init "
-                      + self.tempest_directory + ";fi;cd "
-                      + self.tempest_directory
-                      + ";discover-tempest-config --deployer-input "
-                      "~/tempest-deployer-input.conf --debug --create "
-                      "--network-id " + external_sub_guid
-                      + " object-storage-feature-enabled.discoverability "
-                      "False object-storage-feature-enabled.discoverable_apis"
+        cmd_route = ("source ~/" + setts.overcloud_name + "rc;" +
```

Comment:
I see you fixed this as the "+" was missing in the original.  Did the original work?

Reply:
yes